game/control.js

```diff
@@ -75,7 +75,7 @@ class HexController extends Controller {
     super(
       [
         {
-          direction: HexDirection.UP_LEFT,
+          direction: HexDirection.UP_LEFT, // constructor() from direction.js
```

Comment:
I was still learning about Javascript class functions, so this is just me writing to remind myself...